### ISSUE TITLE
PDOX - NOTIFICATIONS - add project description

### ISSUE THREAD

**Issue by dhenry-KCI** [kcigeospatial/Fred_Co_Land-Management#1518]
Remove the Occupancy Type and Add the Description field (this will show the project name)
See highlighted location.
AR

![image](https://user-images.githubusercontent.com/4129185/62795052-c770ba00-baa3-11e9-9411-91e7c6eca64f.png)

**Comment by megisys** [comment]
IPS Can not Fix this issue. Maybe Project Dox can update the Email Notification

**Comment by dhenry-KCI** [comment]
@RS-Avolve Yes please pull in the Description field to provide additional information for the applicant.

**Comment by RS-Avolve** [comment]
I  changed the email to look like  the following - please note that until an actual project uses this email, the information such as [ProjectName] and [ProjectDescription] will not be populated

![Screenshot Email Template with Project Description](https://user-images.githubusercontent.com/53225350/64446165-536f0500-d095-11e9-8087-c2db45780681.PNG)

**Comment by RS-Avolve** [comment]
I made the change for this email template in both PROD and TEST.  I responded to Github ticket and included a screenshot of how the HTML will look.

NOTE: For now I only added the Project Description field for the Planning workflow Additional Reviews activity.  If this is tested and works as needed,  I can add the Project Description to the other workflow activities using this email template.  You would have to let me know to do that, I do not want to assume this is what you need.

Robert Sosnowski, PMP
Avolve Software | Project Manager
rsosnowski@AvolveSoftware.com<mailto:rsosnowski@AvolveSoftware.com>
Mobile: (208) 577-8070
[cid:image001.png@01D564A3.AEDF8A30]

**Comment by dhenry-KCI** [comment]
the "additional reviews" email still shows the occupancy type 

![image](https://user-images.githubusercontent.com/4129185/65972359-b73be200-e437-11e9-93b2-846536591c69.png)


MW

**Comment by RS-Avolve** [comment]
Unfortunately the Occupancy Type cannot be removed - I made the following comment on Aug 27:
"This cannot be changed; the email template pulls in the Project Name in that location. The project name has been defined on the IPS side as that full string: "PL244949 - PLAT - PUBLIC - NRSMSC". So when the email template pulls in the project name it pulls the entire string in. We cannot pull in just part of the project name."

Regarding adding Description to the email templates for the Applicants - this has been done in both PROD and TEST for the Applicant Upload, Applicant Resubmit, and Prescreen Corrections activities.

**Comment by dhenry-KCI** [comment]
understood

MW/AR
